clang/test/Driver/cxx23-modules-print-library-module-manifest-path.c

```diff
@@ -0,0 +1,15 @@
+// Test that -print-library-module-manifest-path finds the correct file.
+
+// RUN: %clang -print-library-module-manifest-path \
```

Comment:
--print-runtime-dir, --print-target-triple,  --print-supported-cpus, and --print-supported-extensions take 2 dashes.

Reply:
They work with one or two dashes, `Flag<["-", "--"]`, in `Options.td` does that. This option does the same.

Reply:
The trend seems to go to two dashes and there are no tests for 2 dashes.

Reply:
I wasn't aware of that. I got used to using one dash using clang.

Reply:
A lot of new options do support only the `--` form. For `--print-*` options, many support both `-` and `--`. Since it's difficult to remember, I think supporting both for the new option is fine.